scss/components/_search-bar.scss

```diff
@@ -42,6 +42,17 @@ $search-button-width: u(5.6rem);
 
 .search__example {
   margin-top: u(1rem);
+
+  // For aligning the example text after a select input
+  &.search__example--with-select {
+    margin: u(1rem) 0 0 u(21rem);
```

Comment:
Do you want to set this `margin-left` on small screens by default or should it only be on larger screens?